packages.json

```diff
@@ -23487,5 +23487,19 @@
     "description": "Youtube data extractor",
     "license": "MIT",
     "web": "https://github.com/thisago/ytextractor"
+  },
+  {
+    "name": "config",
```

Comment:
that's a pretty generic name for a makefile-clone. Is this definitely the name you want?

Reply:
no, I dont know why the name is config

Reply:
the name I whant is `bs`

Reply:
Your package name is the name of your .nimble file: https://github.com/maubg-debug/build-sys/blob/main/config.nimble

Reply:
aaaaaa

Reply:
Could u just change the name to bs and I will change the file?

Reply:
sry, i didntnknow that

Reply:
sure:

```suggestion
    "name": "bs",
```